app/src/test/java/com/projectit3048c/ss23/CalorieCounterTest.kt

```diff
@@ -29,13 +29,13 @@ class CalorieCounterTest {
     @get:Rule
     var rule: TestRule = InstantTaskExecutorRule()
 
-    lateinit var foodService: FoodService
+    private lateinit var foodService: FoodService
     var allFoods: List<Food>? = ArrayList<Food>()
 
     //lateinit var mvm: MainViewModel
 
     @MockK
-    lateinit var mockFoodService: FoodService
+    internal lateinit var mockFoodService: FoodService
```

Comment:
Added appropriate visibility modifiers to variables per suggestion: https://kotlinlang.org/docs/visibility-modifiers.html#class-members